pkg/email/templates/new_508_request_to_requester.gohtml

```diff
@@ -0,0 +1,25 @@
+<p>
+  We've received your request for 508 testing. You can <a href="{{.RequestLink}}">view your request in EASi</a> at any point.
+</p>
+<p>
+  <b>Next steps</b>
+</p>
+<ol>
+  <li>
+    <p>
+      <b>Fill in the VPAT and Test plan</b>
+    </p>
+    <p>These documents will help the 508 team know what to test and what criteria they should be looking for when testing.</p>
+    <a href="{{.TemplatesLink}}">Download templates of VPAT and Test plan</a>
+  </li>
+  <li>
+    <p>
+      <b>Upload the completed documents to EASi</b>
+    </p>
+    <p>Go to your request page and upload the documents. The 508 team will review these documents and make sure they have a clear understanding of your application and testing needs.</p>
+    <p>
+      <b>Both documents are needed to proceed with your request. If you've already uploaded the documents, please wait for the 508 team to get back to you.</b>
+    </p>
+  </li>
+</ol>
+<p>If you have any questions, contact the 508 team at <a href="mailto:CMS_Section508@cms.hhs.gov">CMS_Section508@cms.hhs.gov</a></p>
```

Comment:
Instead of hardcoding in the CMS_ email address let's pass that in as a parameter alongside RequestLink etc.